chain/messagepool/selection.go

```diff
@@ -97,7 +97,7 @@ func (sm *selectedMessages) tryToAdd(mc *msgChain) bool {
 		sm.msgs = append(sm.msgs, mc.msgs...)
 		sm.blsLimit -= l
 		sm.gasLimit -= mc.gasLimit
-	} else if mc.sigType == crypto.SigTypeSecp256k1 {
+	} else if mc.sigType == crypto.SigTypeSecp256k1 || mc.sigType == crypto.SigTypeDelegated {
```

Comment:
Not related to this particular line, but I feel like after the upgrade we should rename places which handle both delegated and secp messages to `signed` or something like that (including the field in blockheader, which we can rename because it's tuple-serialized).

Reply:
Yep, Signature-bearing or something rather than Signed. (BLS message are also signed, we just aggregate their signatures when we pack them into a block).